### ISSUE TITLE
Link text style and color

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#4000]
Would it be possible to match the link text style and color from SearchWorks? 
- color: F2EFE8
- All links have the dotted line underline
- On hover, color is 00548F and underline is solid
- focus ring is black
